packages/taiko-client/docs/swagger.json

```diff
@@ -0,0 +1,310 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Taiko Preconfirmation Server API",
+    "termsOfService": "http://swagger.io/terms/",
+    "contact": {
+      "name": "API Support",
+      "url": "https://community.taiko.xyz/",
+      "email": "info@taiko.xyz"
+    },
+    "license": {
+      "name": "MIT",
+      "url": "https://github.com/taikoxyz/taiko-mono/blob/main/LICENSE.md"
+    },
+    "version": "1.0"
+  },
+  "paths": {
+    "/healthz": {
+      "get": {
+        "consumes": ["application/json"],
+        "produces": ["application/json"],
+        "summary": "Get current server health status",
+        "operationId": "health-check",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "/preconfBlocks": {
```

Comment:
preconfTransactions

Reply:
Renamed to `POST /tentativeBlocks` as suggested below